salt/metalk8s/roles/etcd/registered.sls

```diff
@@ -0,0 +1,17 @@
+{# Should be run by the orchestrator runner #}
```

Comment:
No, this doesn't belong here. `metalk8s.roles.*` are roles defined for specific minions. This SLS, which is to be executed on the master, belongs... somewhere else. It's not part of the `etcd` node-role.

Reply:
moved